diff --git a/profex/projectWidget/bgmnbackendconfig.cpp b/profex/projectWidget/bgmnbackendconfig.cpp
index 25d9dd5..7ac7f4c 100644
--- a/profex/projectWidget/bgmnbackendconfig.cpp
+++ b/profex/projectWidget/bgmnbackendconfig.cpp
@@ -218,9 +218,6 @@ bool BgmnBackendConfig::guessRepositories()
 int BgmnBackendConfig::guessExecutablesUnix()
 {
     QFileInfo pe(qApp->applicationFilePath());
-    QDir ped(pe.absoluteDir());
-    ped.cdUp();
-    ped.cd("BGMNwin");
 
     int errors = 0;
 
@@ -228,12 +225,12 @@ int BgmnBackendConfig::guessExecutablesUnix()
     // note: the order of scanning determines the priority of the version that will be used
     // if several installations are found
     QVector<QDir> defaultExeLocations;
-    defaultExeLocations << ped.absolutePath()
+    defaultExeLocations << pe.absolutePath()
             << QDir("/opt/Profex-BGMN/BGMNwin")
             << QDir(QString("%1/BGMN").arg(QDir::homePath()))
             << QDir(QString("%1/BGMNwin").arg(QDir::homePath()))
             << QDir("/opt/bgmnwin")
-            << QDir("/opt/bgmn")
+            << QDir("/app/bgmn")
             << QDir("/opt/bgmn-4.2.23")
             << QDir("/opt/bgmn-4.2.22")
             << QDir("/opt/BGMNwin")
@@ -398,15 +395,12 @@ int BgmnBackendConfig::guessExecutablesMac()
 int BgmnBackendConfig::guessRepositoriesUnix()
 {
     QFileInfo pe(qApp->applicationFilePath());
-    QDir ped(pe.absoluteDir());
-    ped.cdUp();
 
     int errors = 0;
 
     QVector<QDir> defaultRepoLocations;
-    defaultRepoLocations << ped
-                         << QDir("/opt/Profex-BGMN/BGMN-Templates")
-                         << QDir("/opt/BGMN-Templates")
+    defaultRepoLocations << QDir("/opt/Profex-BGMN/BGMN-Templates")
+                         << QDir("/app/BGMN-Templates")
                          << QDir(QString("%1/Documents/BGMN-Templates").arg(QDir::homePath()))
                          << createUserRepos();
 
