diff --git a/profex.pro b/profex.pro
index 5266dcd..aa845ea 100644
--- a/profex.pro
+++ b/profex.pro
@@ -7,17 +7,11 @@ SUBDIRS = zlib \
     waterfall \
     electrondensity \
     scantracer \
-    cmdtools/pxanytoxy \
-    cmdtools/pxapplypreset \
-    cmdtools/pxcifindex
 
 profex.depends = zlib quazip libXrdIO
 waterfall.depends = zlib quazip libXrdIO
 electrondensity.depends = zlib quazip libXrdIO
 scantracer.depends = zlib quazip libXrdIO
-pxcifindex.depends = zlib quazip libXrdIO
-pxanytoxy.depends = zlib quazip libXrdIO
-pxapplypreset.depends = zlib quazip libXrdIO
 
 # make these files show up in Qt Creator
 OTHER_FILES += .gitignore \
